src/Microsoft.PowerShell.Commands.Utility/commands/utility/convert-HTML.cs

```diff
@@ -228,6 +228,68 @@ public string[] PreContent
         private string[] _preContent;
 
         /// <summary>
+        /// Sets and Gets the meta property of the HTML head
+        /// </summary>
+        /// <returns></returns>
+        [Parameter(ParameterSetName = "Page")]
+        [ValidateNotNullOrEmpty]
+        public Hashtable Meta
+        {
+            get
+            {
+                return _meta;
+            }
+            set
+            {
+                _meta = value;
+                _metaSpecified = true;
```

Comment:
I believe the typical pattern is to use [InvocationInfo.BoundParameters](https://msdn.microsoft.com/en-us/library/system.management.automation.invocationinfo.boundparameters(v=vs.85).aspx)